docs/developer-environment-setup.md

```diff
@@ -0,0 +1,64 @@
+# Setting up the Local Development Environment
+
+## Prerequisites
+
+Ensure the following are installed:
+
+- [Postgres](https://www.postgresql.org/)
+- [asdf](https://asdf-vm.com/)
+
+## Repository and Version Control
+
+- Ensure you have push access (most likely by being added to the **@bcgov/cas-developers** GitHub team) to [bcgov/cas-cif](https://github.com/bcgov/cas-cif).
+- Ensure you have [GPG commit signing](https://docs.github.com/en/github/authenticating-to-github/signing-commits) set up on your local environment.
+  - First, ensure your `git config user.email` is set to the email address you want to use for signing.
+  - You can verify it's working when you commit to a branch and the signature is indicated by `git log --show-signature`. Once pushed, a "Verified" badge appears next to your commits on GitHub.
+- Clone a local copy of [bcgov/cas-registration](https://github.com/bcgov/cas-registration).
+
+## Backend Environment Setup
+
+1. From the `bc_obps` directory, run `make install_dev_tools`. This will install asdf plugins, poetry and activate the poetry virtual environment.
```

Comment:
```suggestion
1. In both the `client` and `bc_obps` directories, create a `.env` file and copy the contents of `.env.example` file of the respective directory into it. See the 1Password vault for the values.
2. From the `bc_obps` directory, run `make install_dev_tools`. This will install asdf plugins, poetry and activate the poetry virtual environment.
```
`make migrate` won't work if you haven't set up the `.env` file first